docs/repo/property-pages/how-to-add-a-launch-profile.md

```diff
@@ -0,0 +1,104 @@
+
+![New Launch Profile](new-launch-profile.png)
+
+## Option 1: XAML file on disk
+
+In this option, the XAML resides in a standalone .xaml file on disk and is included in the end user's project as a specific kind of MSBuild item, `PropertyPageSchema`. Visual Studio reads these items from the project to determine which kinds of Launch Profiles it can display.
+
+This may be an attractive option if you already have a NuGet package or Visual Studio extension that injects MSBuild .props and .targets files into the end user's project, or if you want to use MSBuild `Condition`s to control when the Launch Profile UI is available to a project. 
+
+### Step 1 (optional): Add the Microsoft.Build.Framework package
+
+Use the NuGet Package Manager to add the Microsoft.Build.Framework package to your project. This is an optional step, but it will allow the XAML editor to find the [Rule](https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.framework.xamltypes.rule) type (and related types) and provide code completion, tool tips, Go to Definition, and other features while you type.
+
+### Step 2: Define the XAML file
+
+Add a new XAML file named "MyLaunchProfile.xaml" to your project. Depending on how the file is created you may end up with a `<Page>` item in your project but this is not what we want as we're not using the file to describe a piece of WPF UI.
+
+Update your project to replace the `<Page>` item with one of the following:
+
+- SDK-style projects:
+  ``` xml
+  <None Update="MyPropertyPage.xaml">
+    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+  </None>
+  ```
+- Non-SDK-style projects:
+  ``` xml
+  <None Include="MyPropertyPage.xaml">
+    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+  </None>
+  ```
+
+Now VS won't do anything with this file but copy it to the output directory when you build.
+
+### Step 3: Define the `PropertyPageSchema` item
+
+Next you need to update the .props or .targets files imported by the end users' projects to properly reference the property page so Visual Studio can find it. Note that the creation and distribution of the .props and .targets files (as well as the distribution of MyPropertyPage.xaml itself) is beyond the scope of this document.
+
+Add the following item to your .props or .targets file:
+
+``` xml
+<PropertyPageSchema Include="path\to\MyLaunchProfile.xaml">
+  <Context>Project</Context>
+</PropertyPageSchema>
+```
+
+### Step 4: Describe the property page
+
+Replace the contents of MYlaunchProfile.xaml with the following:
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<Rule Name="MyLaunchProfile"
+      Description="Properties associated with launching and debugging a custom debug target."
+      DisplayName="MyCustomDebugTarget"
```

Comment:
Is this actually a display string? Adding spaces would make that more clear, to me at least, that this is not a machine-interpreted identifier (unlike the CommandName).

```suggestion
      DisplayName="My Custom Debug Target"
```